### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

Misc fixes

Fixes # (issue)

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/pages/event-types/[type]/index.tsx [hunk 1]
```diff
@@ -304,6 +304,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
             seatsShowAttendees,
             metadata,
             customInputs,
+            userId: 4,
```
Comment (PeerRich) [inline apps/web/pages/event-types/[type]/index.tsx:307]: what is this? hardcoded id looks suspicious

File: packages/trpc/server/routers/viewer/eventTypes.tsx [hunk 2]
```diff
@@ -448,8 +448,15 @@ export const eventTypesRouter = router({
...
       id,
       hashedLink,
+      // Extract this from the input so it doesn't get saved in the db
+      // eslint-disable-next-line
+      userId,
+      // eslint-disable-next-line
+      teamId,
       ...rest
     } = input;
+    console.log("🚀 ~ file: eventTypes.tsx:458 ~ update:eventOwnerProcedure.input ~ schedule", schedule);
```
Comment (PeerRich) [inline packages/trpc/server/routers/viewer/eventTypes.tsx:458]: same here, looks like debugging?

**Review by CarinaWolli — DISMISSED** [review]

File: packages/trpc/server/routers/viewer/workflows.tsx [hunk 3]
```diff
@@ -926,6 +926,37 @@ export const workflowsRouter = router({
...
+          id: eventTypeId,
+          users: {
+            some: {
+              id: ctx.user.id,
+            },
+          },
+        },
+      });
+
+      if (!userEventType)
```
[the remark this answers is not in the document]
Comment (CarinaWolli) [reply packages/trpc/server/routers/viewer/workflows.tsx:941]: Maybe this help. I think we do the same here: https://github.com/calcom/cal.com/blob/53d160e49539eaab5395a6fe51d1d7ec03c0c62e/packages/trpc/server/routers/viewer/workflows.tsx#L301

File: packages/trpc/server/routers/viewer/workflows.tsx [hunk 4]
```diff
@@ -926,6 +926,37 @@ export const workflowsRouter = router({
...
+
+      // Check that the workflow belongs to the user
+      const eventTypeWorkflow = await ctx.prisma.workflow.findFirst({
+        where: {
+          id: workflowId,
+          activeOn: {
+            some: {
+              eventTypeId: eventTypeId,
+            },
+          },
```
[the remark this answers is not in the document]
Comment (joeauyeung) [reply packages/trpc/server/routers/viewer/workflows.tsx:952]: Removed, good catch

File: packages/trpc/server/routers/viewer/workflows.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (joeauyeung) [reply packages/trpc/server/routers/viewer/workflows.tsx:941]: That user should still be under the `users` field for team event types, right? Won't this consider that? If the user is not a part of that team event type, I don't think they should be able to edit workflows

**Review by JeroenReumkens — APPROVED** [review]

**Review by CarinaWolli — APPROVED** [review]

**Review by pumfleet — APPROVED** [review]

**Pull request merged**